### PR TITLE
Refining split

### LINKED ISSUE #18: Some original code was NOT exported here - to revert to original code from split
I was checking exports and a large chunk of code is missing here. this is high priority

### PR DESCRIPTION
**Pull request by Melkiades**
Fixes #18

### CODE REVIEW

**Comment by Melkiades** [comment]
I have read the CLA Document and I hereby sign the CLA

File: R/export_as_docx.R [hunk 1]
```diff
@@ -74,7 +18,7 @@ formatters::export_as_pdf
 #' @param ... (`any`)\cr additional arguments passed to [tt_to_flextable()].
 #'
 #' @note `export_as_docx()` has few customization options available. If you require specific formats and details,
-#'   we suggest that you use [tt_to_flextable()] prior to `export_as_docx`. Only the `title_as_header` and
+#'   we suggest that you use [tt_to_flextable()] prior to `export_as_docx`. Only the `titles_as_header` and
```
Comment (shajoezhu) [inline R/export_as_docx.R:21]: good catch

File: R/package.R [hunk 2]
```diff
@@ -5,9 +5,6 @@
 #' @importFrom magrittr %>%
 #' @importFrom stats setNames
 #' @import methods
-#' @import formatters
-#' @import rtables
```
Comment (shajoezhu) [inline R/package.R:9]: these are needed

**Review by shajoezhu — CHANGES_REQUESTED** [review]
hi @Melkiades , can you add formatters for rtables :: to 

all_footers all_titles collect_leaves coltree decimal_align font_spec
  make_row_df matrix_form mf_aligns mf_nlheader mf_spans mf_strings
  obj_name paginate_table propose_column_widths spans_to_viscell
  value_names

thanks!

**Review by shajoezhu — APPROVED** [review]
lgtm! thanks @Melkiades

**Pull request merged**